admin_manual/issues/general_troubleshooting.rst

```diff
@@ -171,7 +171,7 @@ Web server and Nextcloud itself.
    Linux distros or operating systems they can differ.
 
 * The logfile of Apache2 is located in ``/var/log/apache2/error.log``.
-* The logfile of PHP can be configured in your ``/etc/php/8.0/apache2/php.ini``.
+* The logfile of PHP can be configured in your ``/etc/php/8.1/apache2/php.ini``.
```

Comment:
lets use 8.3 to avoid touching it every year? :P

Reply:
Done